### PR TITLE
Split init route to init & information endpoints

### PR DESCRIPTION
**Pull request by Convly**

### CODE REVIEW

**Review by alexandrebodin — CHANGES_REQUESTED** [review]
a Unit test ?

File: packages/strapi-admin/controllers/__tests__/admin.test.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+// const createContext = require('../../../../test/helpers/create-context');
```
Comment (petersg83) [inline packages/strapi-admin/controllers/__tests__/admin.test.js:3]: Should we remove this comment?

**Review by petersg83 — CHANGES_REQUESTED** [review]
A little remark

File: packages/strapi-admin/controllers/__tests__/admin.test.js (hunk 1 above)
Comment (Convly) [reply packages/strapi-admin/controllers/__tests__/admin.test.js]: Thanks, fixed :tada:

**Review by petersg83 — APPROVED** [review]
LGTM :)

**Pull request merged**